### PR TITLE
2024 tips update

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

New year, new tips.

## Why It's Good For The Game

You know the drill, and so do I.

## Changelog

:cl: the TGMC community
spellcheck: 2024 tips update
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
I would like the community to submit tips. Help players out with some protips!

File: strings/tips/marine.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Go to Vahalla to experiment with loadouts and weapons. You don't reset your respawn time in Vahalla!
```
Comment (iamafsh) [inline strings/tips/marine.txt:1]:
```suggestion
Go to Valhalla to experiment with loadouts and weapons. You don't reset your respawn time in Valhalla!
```

**Comment by Wirpy** [comment]
1# TGMC Lore drop;
didu know 
they moth sax
on tg 
se 
sex

2# Tip:
ban evade
d
im in

3# Tip:
Keep your moth saxes safe, wear protection

4# Help wanted:
how do if get unbanned asd amin

5# TGMC Lore drop;
Lichnarök is a real and canon event in TGMC lore, consult lore masters for further information

6# TGMC Lore drop;
TGMC, the only server where an admin attempted grand larceny of a freddy fazbear head

7# TGMC Lore drop;
harpidi toilet

8# Tip:
You can change colour as a xenomorph by interacting with the silo

9# Tip: 
Imagine how stupid the average marine is then realize half of all marines are stupider than that

10# Tip:
Seek the wisdom of the curve, It will grant thee thy enlightment thous'th seek 

11# Tip: 
Use smelling salts to increase your performance, beware of fiery computers!

12# 
real

13# Tip:
Sexoids aren't real, grow up. Sectoids on the other hand are real and will mindblast you.

14# 
moth sax

15#  
TGMC is fucking horrible and here are all the reasons why

1: The staff are so fucking stupid, especially that Generic DM, all they all do all day is have gay xeno sex and deny us players the things we not only want but DESERVE (more crayons)

2: The players are also so fucking stupid, they are all either ungas or crayons eaters and there is no inbetween, all they do is rush to the front line and die

3: Codebase bad, so much shit code and bad PRs, xeno respawn rework is dumb and they salivated over it for WEEKS but there have been nothing on the PRESSING MATTERS we as players have asked for for YEARS (more crayons!!!!)

4: They have no forum you can use to tell them how dumb they are, all they have is stupid gaycord and gitgay

5: Too many ANIME

6: Fuck you TGMC you smell like pile of crayons

**Comment by Helg2** [comment]
You could add something like `Remember! Dying is gay!` to the meme category

**Comment by 1kajowsmol** [comment]
You can roll in snow, or walk into water to put out fire instantly. (some people especially don't know the snow part)

**Comment by Wisemonster** [comment]
If meme suggestions are allowed, may I suggest "Alert! There is a red xeno in the FOB!"? For a more serious suggestion, a tip saying "If you are low on blood, eating food can help you replenish missing blood."

**Comment by yyzsong** [comment]
**Xeno:**
as a ravager, rage mode will make you stagger immune, reset your armour, give you additional fury and boost your speed for a time. Use it as an escape tool, especially if you're too low on fury to endure.
as a ravager, stepping on a tile of fire will give you 50 fury. This has a one second cooldown between uses.
as a ravager, don't get overconfident.
as a ravager, you will regenerate your fury, up to a point.

as a hunter, entering stalk/walk mode while stealthing will make you more transparent. 
remove the tip about standing still. I'm pretty sure that doesn't do anything

as a caste with acid, you can click directly on a lower-level acid puddle to upgrade it.

as a warlock, your offensive abilities can be cast through glass. Give laser-using marines a taste of their own medicine.

as a gorger, your drain ability can be used on corpses to heal incredibly fast, and scales up with pheromones.
as a gorger, you have very little armour. You do not have to consider sunder as much as other castes.
as a gorger, overheal will let you drain corpses even if you're taking damage! This also applies to anyone else that you've overhealed.
as a gorger, you may psy drain for free.

as a ravager or gorger, you will not be affected by tanglefoot or the tesla rifle. However, pepperballs and the smartgunner's tanglefoot DMR ammo still will.

as a shrike, queen, or king, hijacking the alamo will kill every xeno that you leave groundside, as well as destroying every structure and resetting your psy count. However, dead xenos will be refunded as burrowed larva. Call burrowed after a hijack!

rounies aren't real. Stop asking.
rounies are real. Don't let them tell you otherwise.

remove xeno tip 23 about camping fob before defenses are set. that's a CM thing

**Comment by TheUSSWatercooler** [comment]
Spaceacillin loses effectiveness against severe wound infections. Polyhexanide will pick where it left off at, at the cost of some toxin damage and drowsiness.

You can use bicardine in vali weapons to heal brute on corpses.

Inaprovaline, Russian Red, and Neuraline injections can quickly bring a marine from death's door to walking.

Wash your hands.

**Comment by Sirsnek15** [comment]
Marine: An entire MRE can fit in your boot.

File: strings/tips/xeno.txt [hunk 2]
```diff
@@ -1,7 +1,13 @@
+You can replace normal weed pods with sticky/resting weed pods, but cannot do the inverse.
+Stagger stops most xenomorphs from using abilities. Watch out for weapons that can stagger!
+Different castes have different pheromone strengths, with drone being the lowest and king being the highest.
+Pheromones do not stack, only the strongest applies.
+When placing resin walls, they start at 40% their max health then gain health over time until hitting their max health. For hivelords, they are start at 70%.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:5]:
```suggestion
When placing resin walls, they start at 40% their max health then gain health over time. Walls made by hivelords start at 70% health.
```

File: strings/tips/xeno.txt [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
+Marines suffer no movement penalty for moving on standard or resting weeds, however sticky weeds slow them down significantly.
 As a xeno, you can freely melt barricades while in boiler or defiler gas as long as you can click on the right spot. The gas is also a really good hiding spot to launch ambushes from without marines being aware of you.
 As a xeno, always thank your drones and hivelords for supporting the hive!
-As a xeno, you can drag any item, but you must wait through an actionbar before you can do so!
-As a xeno, do not fight as a young xenomorph.
+Robot healthbars are not directly one to one with their actual health. they will still have some fight left in them after it shows their health has hit critical.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:9]:
```suggestion
Robot healthbars are not directly one to one with their actual health. They will still have some fight left in them after it shows their health has hit critical.
```

File: strings/tips/xeno.txt [hunk 4]
```diff
@@ -10,36 +16,50 @@ As a xeno, it is faster to re-fill an empty acid well than rolling on the ground
 As a xeno, acid wells extinguish fire and explode into an acid cloud when destroyed. The Queen, the Shrike, praetorians, boilers, and hivelords can make acid wells.
 The Queen, the King, the Shrike, and hivelord have access to Queen's Blessing, a menu for the hive to use psychic points to give xenomorphs a cutting edge in the battlefield.
 If you have acid spit, you can spit at a welding fuel tank to blow it up. If the welding fuel is near hostiles, you might give them a pleasant surprise!
+If something comes up in real life while you’re a xenomorph, head back to the silo and use the “ghost” command to let others take over your body. Your name will be cleared
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:19]:
```suggestion
If something comes up in real life while you’re a xenomorph, head back to the silo and use the “ghost” command to let others take over your body.
```

File: strings/tips/xeno.txt [hunk 5]
```diff
@@ -10,36 +16,50 @@ As a xeno, it is faster to re-fill an empty acid well than rolling on the ground
...
 As a carrier, you can hit a host directly with a thrown facehugger to have a high chance of infecting them near instantly. This will take time, however.
 As a carrier, you can hide traps beneath certain objects and items.
+As a carrier you can take out huggers from eggs and replace them with specialized huggers instead, such as a neurotox hugger.
 As a hunter, you have almost perfect stealth if you stand still.
+As a hunter, entering stalk/walk mode while stealthing will make you more transparent.
+As a hunter, a second activation of your mirages ability will exchange your place with one of your mirages! Use this to get back in the fight, when marines start to ignore your mirages.
 As a ravager or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it. As the King, however, you're fireproof until your armor is damaged by sunder.
+As a ravager, rage mode will make you stagger immune, reset your armour, give you additional fury and boost your speed for a time. Use it as an escape tool, especially if you're too low on fury to endure.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:46]:
```suggestion
As a ravager, rage mode will make you stagger immune, reset your armour, give you additional fury and boost your speed for a time. Use it as an escape tool, especially if you're too low on rage to endure.
```

File: strings/tips/xeno.txt [hunk 6]
```diff
@@ -10,36 +16,50 @@ As a xeno, it is faster to re-fill an empty acid well than rolling on the ground
...
 As a spitter, boiler, praetorian, or the Queen, take advantage of unguarded barricades and defenses; melt or constantly spit acid on them if you can.
+As a ravager, stepping on a tile of fire will give you 50 fury. This has a one second cooldown between uses.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:48]: It's just a passive increase now while you're on fire

File: strings/tips/xeno.txt [hunk 7]
```diff
@@ -10,36 +16,50 @@ As a xeno, it is faster to re-fill an empty acid well than rolling on the ground
...
+As a ravager, don't get overconfident.
+As a ravager, you will regenerate your fury, up to a point.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:50]:
```suggestion
As a ravager, you will regenerate your rage, up to a point.
```

File: strings/tips/xeno.txt [hunk 8]
```diff
@@ -56,6 +76,11 @@ As a xeno, attack between the delay of a shotgun being fired.
 As a xeno, listen to the sound of your enemy's magazine! If it sounds hollow, it will soon be empty.
 As a xeno, strike hard and fast.
 As a xeno, you are not disposable. However, sacrificing your life to save the Queen is widely respected.
+Hunter has AP and knockdown when striking from stealth, which can be chained with pounce for a longer stun.
+As a hunter you can swap places with one of your mirages by pressing the button again.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:80]: Duplicate tip

File: strings/tips/xeno.txt [hunk 9]
```diff
@@ -64,3 +89,4 @@ As a boiler, you'll begin to glow with the more globules you have stored.
 If you see a red dot, RUN!
 Allowing marines to push caves only to block off their exit is hillariously effective.
 As a xeno, a Tier 3 unit takes up both a Tier 3 and Tier 2 slot.
+In crash, marines have landed when nuke disks appear on the map.
```
Comment (ivanmixo) [inline strings/tips/xeno.txt:92]: Ehhhhh this is really a bug in my opinion and not an actual feature. I'd leave it out

File: strings/tips/marine.txt [hunk 10]
```diff
@@ -81,12 +96,26 @@ As an engineer, det packs can be set on demolition mode for an explosive trap.
 As an engineer, you build barricades the fastest.
 As an engineer, repair the mining wells. These mining machines will mine phoron or platinum that can be used for requisitions, earning them more points!
 As an engineer, placing turrets on open spaces and without defenses is a poor choice.
-As a corpsman, the medevac stretcher needs to be linked to an active, powered beacon for it to teleport the patient.
+You can use a Terra experimental cell as a discount battery!
+You can upgrade metal barricades with metal in hand!
+Bring at least 1 cell and 1 power control modules to fix APC for disks or intel.
+Acid Upgrade prevents xenos from spewing acid all over undefended barricade.
```
Comment (ivanmixo) [inline strings/tips/marine.txt:102]:
```suggestion
The caustic upgrade prevents xenos from spewing acid all over an undefended barricade.
```

File: strings/tips/marine.txt [hunk 11]
```diff
@@ -1,16 +1,26 @@
+Go to Valhalla to experiment with loadouts and weapons. You don't reset your respawn time in Valhalla!
+Fire disables pheromones, use this information to your advantage.
+Catch larval facehuggers by having an empty hand and throw mode on.
+Standing in a corner on harm intent stops you from behing shuffled, unless you're a marine and a xenomorph on help intent walks into you.
+Jumping will allow you to throw grenades farther, and over allies. Yeet those grenades to xenomorphs!
+Hunger will slow you down. A single protein bar will fill many, if not every, people. Take an extra protein bar too much and you become slower (ironic) and fat.
```
Comment (ivanmixo) [inline strings/tips/marine.txt:6]: will fill many, if not every, people

lol i get its funny but it was removed for a reason

File: strings/tips/marine.txt [hunk 12]
```diff
@@ -1,16 +1,26 @@
...
+If you are low on blood, eating food can help you replenish missing blood.
```
Comment (ivanmixo) [inline strings/tips/marine.txt:7]:
```suggestion
If you are low on blood, eating food can help you replenish blood.
```

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: strings/tips/marine.txt [hunk 13]
```diff
@@ -1,16 +1,26 @@
...
+You can refill the marine jetpack with flamer tanks.
+Jetpacks can fly over most things. Use them to escape your poor decisions of overextending.
 If the main push is stagnating and the area is congested, try to coordinate with a few others and go to a different area.
 When pushing into an area, check the flanks. Especially so in caves and enclosed spaces.
 Resin structures are weaker to melee than they are to bullets. Take out your knives, swords and bayonets!
 Always clear weeds and alien structures as you advance. A resin maze forces marines to engage in CQC against xenomorphs, which decreases your survival rate. That said, this increases your PB chances!
 Always check for traps beneath loose objects on weeds - it could save your FACE!
 As a human, you can do almost anything if you put your mind to it; it just takes time.
-You can link plasteel barricades together using a crowbar. Click one, and then click one adjacent. Repeat as needed.
+Any marines can link plasteel barricades together using a crowbar. Click one, and then click one adjacent. Repeat as needed.
```
[the remark this answers is not in the document]
Comment (ivanmixo) [reply strings/tips/marine.txt:23]:
```suggestion
Any marine can link plasteel barricades together using a crowbar. Click one, and then click one adjacent. Repeat as needed.
```

File: strings/tips/marine.txt [hunk 14]
```diff
@@ -45,10 +55,14 @@ To throw a grenade, Activate (Default: Z) the grenade while in the active hand.
 Quickly store items to a container (bags, satchels, belts, pouches) by pressing the Quick Equip (Default: E) key while holding an item and having a container open.
 Press Quick Equip (Default: E) while you are not holding an item to the active hand to draw the weapon from whatever preferred slot (Default: Suit Storage, can be customized at the Preferences tab) you selected, otherwise you will pull out any item from a container.
 Stasis bags do not pause xenomorph infection entirely, they only slow progress by a lot.
-Stasis bags prevents the DNR (do not revive) timer from ticking, meaning that if corpsmen have a lot of patients, they can prevent brain death via stasis bags.
+Stasis bags prevents brain death timer from ticking, meaning that if corpsmen have a lot of patients, they can prevent brain death via stasis bags.
```
[the remark this answers is not in the document]
Comment (novaepee) [reply strings/tips/marine.txt:58]: DNR is specific in that the player intentionally op out to not get revived. Brain death is not intentional on the player's part. I know we use DNR and brain death as the same, though I prefer that we don't mix these up.

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**